packages/shorebird_cli/lib/src/shorebird_process.dart

```diff
@@ -113,23 +113,40 @@ class ShorebirdProcess {
   }
 }
 
+class ShorebirdProcessResult {
```

Comment:
Why?  Was it sealed and thus couldn't be mocked?

Reply:
It’s now final so it can’t be implemented.